.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from:
+    - .rubocop_todo.yml
```

Comment:
Are you sure about it?
Without renaming I can just run `rubocop` command from root of project and it will use this config file, but if i rename - I should run `rubocop -c .rubocop_rubyzip.yml`